pyrobosim/pyrobosim/manipulation/grasping.py

```diff
@@ -0,0 +1,182 @@
+""" Grasping utilities. """
+
+from ..utils.pose import Pose
+
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
How much effort are we saving with one letter?   ;)

```suggestion
import matplotlib.pyplot as plot
```

Reply:
This is just the "unwritten" standard in all matplotlib examples: https://matplotlib.org/stable/tutorials/introductory/pyplot.html

It's like saying `import numpy as np`.